config.yaml

```diff
@@ -112,3 +112,34 @@ languages:
           url: https://github.com/go-gitea/
           weight: 60
           pre: github
+
+  pt-br:
+    weight: 3
+    languageName: Português brasileiro
```

Comment:
@tboerger @andreynering and if we used country flags instead name like I make in my projects?

![gitea_country_flags](https://cloud.githubusercontent.com/assets/1520407/21457016/4a93adc8-c913-11e6-885c-4e2b21c46160.png)